### ISSUE TITLE
strokeDasharray on Line no longer working

### ISSUE THREAD

**Issue by pcmaffey** [recharts/recharts#318]
Updated to 0.16 this morning and noticed that now the prop strokeDasharray on Line component doesn't get passed down to Curve component. "7 11" outputs as "0px, 0px"

**Comment by matthewberkompas** [comment]
@xile611, in my project I have a line chart with an input above it to enable users to enter a goal number which will then appear on the chart as a dashed line using `strokeDashArray`. I just updated to version 0.16.2 which was intended to fix the problem @pcmaffey mentioned above (and it did for the most part). I'm running into an issue, however, where whenever I enter a goal number that is 2 digits long (it doesn't do this with 1 or 3+ digit numbers), the goal line is no longer dashed and the console throws this error:
```
Cannot assign to read only property 'length' of object '#<CSSStyleDeclaration>'  
```
I confirmed that this _is_ an issue with version 0.16.2. Any ideas? Thanks a lot.
